src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -278,4 +278,13 @@ public void setDatastorePath(File datastorePath) {
   public void setEnvironment(Map<String, String> environment) {
     this.environment = environment != null ? ImmutableMap.copyOf(environment) : null;
   }
+
+  @Override
+  public File getWorkingDirectory() {
+    return workingDirectory;
+  }
+
+  public void setWorkingDirectory(File workingDirectory) {
+    this.workingDirectory = workingDirectory;
```

Comment:
This class doesn't do any validation by its design (a simple bean), so if we were to check it, we should do it in other places. But then, `java.lang.Process` will ultimately throw `IOException`, so I'm not inclined to add an extra check.